### PR TITLE
[ACA-5393] Fix broken gateway_api lookup plugin

### PR DESCRIPTION
**Pull request by rohitthakur2590**
## Description

- What is being changed?
   tests being changed  to ensure we replace builtin_uri with module exist operation
- Why is this change needed?
 builtin uri bypass the http connection
- How does this change address the issue?

## Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Test update
- [ ] Refactoring (no functional changes)
- [ ] Development environment change
- [ ] Configuration change

## Self-Review Checklist

- [ ] I have performed a self-review of my code
- [ ] I have added relevant comments to complex code sections
- [ ] I have updated documentation where needed
- [ ] I have considered the security impact of these changes
- [ ] I have considered performance implications
- [ ] I have thought about error handling and edge cases
- [ ] I have tested the changes in my local environment

### Steps to Test
1. 
2. 
3. 

### Required Actions

- [ ] Requires documentation updates
  
- [ ] Requires downstream repository changes
  
- [ ] Requires infrastructure/deployment changes
  
- [ ] Requires coordination with other teams
  
- [ ] Blocked by PR/MR: #XXX

### CODE REVIEW

File: tests/integration/targets/users_test/tasks/main.yml [hunk 1]
```diff
@@ -131,28 +131,6 @@
         that:
           - joe_superuser is changed
 
-    - name: Give Timmy auditor
```
Comment (rohitthakur2590) [inline tests/integration/targets/users_test/tasks/main.yml:134]: its getting removed , that why these are going away

File: tests/integration/targets/role_team_assignments_test/tasks/main.yml [hunk 2]
```diff
@@ -145,23 +145,17 @@
...
-          - "assignment_query.json.count > 0"
-        fail_msg: "No role assignment found for Custom Role ID {{ custom_role.id }} and Team ID {{ team1.id }}."
+    - name: Verify role team assignment exists for Team 1 and Custom Role
+      ansible.platform.role_team_assignment:
+        role_definition: "{{ custom_role_name }}"
+        team: "{{ team1.team.name }}"
+        assignment_objects:
+          - name: "{{ org1.organization.name }}"
+            type: "organizations"
+        state: exists
```
Comment (rohitthakur2590) [inline tests/integration/targets/role_team_assignments_test/tasks/main.yml:158]: replacing raw_uri as it was bypassing http connection

File: plugins/plugin_utils/docs/user.py [hunk 3]
```diff
@@ -52,21 +52,6 @@
...
-    aliases: ['auditor']
-
-  organizations:
-    description:
-      - List of organization names to associate with the user
-      - Organizations must already exist
-      - Deprecated - use role_user_assignment module instead
-    type: list
-    elements: str
-
```
Comment (rohitthakur2590) [inline plugins/plugin_utils/docs/user.py:69]: this is deprecated and to be removed

**Review by nickbhasin — APPROVED** [review]
Can you add JIRA number to PR title ?
